framework/db/mapping_manager.py

```diff
@@ -74,6 +74,13 @@ def GetMappingTypes(self):
         """
         return self.mapping_types
 
+    def GetAllMappings(self):
```

Comment:
In addition, a quick docstring to document the purpose of the function would be welcome! :)